packages/next/plugins/with-nx.ts

```diff
@@ -204,6 +207,22 @@ function withNx(
         dependencies = result.dependencies;
       }
 
+      // For Next.js 13.1 and greater, make sure workspace libs are transpiled.
+      forNextVersion('>=13.1.0', () => {
```

Comment:
Added a util function that will us to support different configuration that are introduced in different Next.js versions.